tests/test_local_settings.py

```diff
@@ -149,3 +161,14 @@ def test_pod_mutation_hook(self):
             settings.pod_mutation_hook(pod)
 
             assert pod.namespace == 'airflow-tests'
+
+    def test_custom_policy(self):
+        with SettingsContext(SETTINGS_FILE_MUST_HAVE_OWNER_POLICY, "airflow_local_settings"):
+            from airflow import settings
+            settings.import_local_settings()
+
+            task_instance = MagicMock()
+            task_instance.owner = 'airflow'
+            with self.assertRaises(AirflowClusterPolicyViolation):
+                settings.task_must_have_owners(task_instance)  # pylint: disable=no-member
```

Comment:
The ``task_must_have_owners`` attribute is not expected to exist on this object. We should only test `cluster_policy` in our tests. 

For now, this test only tests if the function is copied correctly by ``import_local_settings``, but it's already covered by other tests - `test_pod_mutation_hook`,  `test_policy_function`